swingtix/bookkeeper/models.py

```diff
@@ -247,7 +247,7 @@ def natural_key(self):
 
     account = models.ForeignKey(Account, db_column='accid', related_name='entries')
 
-    amount = models.DecimalField(max_digits=8, decimal_places=2,
+    amount = models.DecimalField(max_digits=1000, decimal_places=2,
```

Comment:
What do you think about setting it to 16 instead of 1000?  This would make it small enough to fit in a 64bit integer.
(64 bits is enough for a signed 18 digit number, minus 2 decimal places is 16.)

Is that enough?